### DISCUSSION TITLE
Idea: Go to center button

### DISCUSSION THREAD

**Discussion by AstralButOnGitHub** [ScratchAddons/ScratchAddons#5223] [Ideas]
![Untitled](https://user-images.githubusercontent.com/116039257/196302759-73de3526-17d4-46d6-8fd2-00707d6480be.png)
Pressing the button would take the selected content and move it to the center

**Comment by Endermaster3690** [comment]
off topic but just
![Screenshot 2022-10-17 184248](https://user-images.githubusercontent.com/80355301/196305107-2bebcfa7-26d4-4df2-b8e4-d03bb99ceb9e.png)
why did you put 2 upvote reactions on your own post

**Reply by AstralButOnGitHub** [reply]
Oh thats what that means
(I'm new to github)

**Comment by BroJac5246** [comment]
I don't really know if that'd be worth it, though, since it really isn't that much of a pain to just manually center it.

**Reply by AstralButOnGitHub** [reply]
I guess
